packages/sns/src/converters/governance.converters.spec.ts

```diff
@@ -8,106 +11,136 @@ describe("governance converters", () => {
+      const default_followees: DefaultFollowees = {
+        followees: [[BigInt(20), { followees: [{ id: new Uint8Array(0) }] }]],
+      };
+      const max_dissolve_delay_seconds = BigInt(21);
+      const max_dissolve_delay_bonus_percentage = BigInt(22);
+      const max_followees_per_function = BigInt(23);
+      const neuron_claimer_permissions = { permissions: new Int32Array() };
+      const neuron_minimum_stake_e8s = BigInt(24);
+      const max_neuron_age_for_age_bonus = BigInt(25);
+      const initial_voting_period_seconds = BigInt(26);
+      const neuron_minimum_dissolve_delay_to_vote_seconds = BigInt(27);
+      const reject_cost_e8s = BigInt(28);
+      const max_proposals_to_keep_per_action = 2;
+      const max_number_of_neurons = BigInt(29);
+      const transaction_fee_e8s = BigInt(30);
+      const max_age_bonus_percentage = BigInt(31);
+      const neuron_grantable_permissions = { permissions: new Int32Array() };
+      const final_reward_rate_basis_points = BigInt(32);
+      const initial_reward_rate_basis_points = BigInt(33);
+      const reward_rate_transition_duration_seconds = BigInt(34);
+      const round_duration_seconds = BigInt(35);
+      const max_number_of_principals_per_neuron = BigInt(2);
+
       const action: ActionCandid = {
         ManageNervousSystemParameters: {
-          default_followees: [
-            {
-              followees: [
-                [BigInt(2), { followees: [{ id: new Uint8Array(0) }] }],
-              ],
-            },
+          default_followees: [default_followees],
+          max_dissolve_delay_seconds: [max_dissolve_delay_seconds],
+          max_dissolve_delay_bonus_percentage: [
+            max_dissolve_delay_bonus_percentage,
+          ],
+          max_followees_per_function: [max_followees_per_function],
+          neuron_claimer_permissions: [neuron_claimer_permissions],
+          neuron_minimum_stake_e8s: [neuron_minimum_stake_e8s],
+          max_neuron_age_for_age_bonus: [max_neuron_age_for_age_bonus],
+          initial_voting_period_seconds: [initial_voting_period_seconds],
+          neuron_minimum_dissolve_delay_to_vote_seconds: [
+            neuron_minimum_dissolve_delay_to_vote_seconds,
           ],
-          max_dissolve_delay_seconds: [BigInt(2)],
-          max_dissolve_delay_bonus_percentage: [BigInt(2)],
-          max_followees_per_function: [BigInt(2)],
-          neuron_claimer_permissions: [{ permissions: new Int32Array() }],
-          neuron_minimum_stake_e8s: [BigInt(2)],
-          max_neuron_age_for_age_bonus: [BigInt(2)],
-          initial_voting_period_seconds: [BigInt(2)],
-          neuron_minimum_dissolve_delay_to_vote_seconds: [BigInt(2)],
-          reject_cost_e8s: [BigInt(2)],
-          max_proposals_to_keep_per_action: [2],
+          reject_cost_e8s: [reject_cost_e8s],
+          max_proposals_to_keep_per_action: [max_proposals_to_keep_per_action],
           wait_for_quiet_deadline_increase_seconds: [],
```

Comment:
No value here?

Reply:
no, I want to test that some return `undefined` also.